Misc/NEWS.d/next/Library/2022-07-25-15-45-06.gh-issue-95231.i807-g.rst

```diff
@@ -0,0 +1,3 @@
+Handle EPERM errors gracefully in Lib/crypt.py
+One of the scenarios we hit this condition is when kernel fips is enabled
+and trying to use md5 from crypt module.
```

Comment:
How about this? cc. @tiran 

```suggestion
Fail gracefully if :data:`~errno.EPROM` or :data:`~errno.ENOSYS` is raised when loading
:mod:`crypt` methods. This may happen when trying to load ``MD5`` on a Linux kernel
with :abbr:`Federal Information Processing Standard (FIPS)` enabled.
```